Simplenote/Information/NoteInformationViewController.swift

```diff
@@ -319,6 +336,7 @@ private struct Localization {
     static let information = NSLocalizedString("Information", comment: "Card title showing information about the note (metrics, references)")
```

Comment:
I believe this string is no longer used and can be removed.

Reply:
I wasn't sure if we wanted to remove that or not.  Removed it